rust-mode.el

```diff
@@ -198,6 +198,18 @@ to the function arguments.  When nil, `->' will be indented one level."
   :safe #'booleanp
   :group 'rust-mode)
 
+(defcustom rust-format-show-buffer nil
```

Comment:
Why not set the default to t ? I think most users want the buffer the be displayed when there's an error ?

Reply:
I think that the idea is to show the error in the echo area by default.

Reply:
I see.